packages/translate/__test__/elf-translate-element.test.js

```diff
@@ -1,9 +1,16 @@
 import { fixture, expect, isIE, elementUpdated, nextFrame } from '@refinitiv-ui/test-helpers';
```

Comment:
There is not need to change tests for `translate`. To start with, we do not test 3rd party software (format.js polyfills in this case), but just ensure that the translation package works. And then, if you want to include these, I can see that translations are checked for Russian locale, which is not included.

Reply:
Already reverted

Reply:
OK, I already reverted the change.